src/geocel/vg/VecgeomTrackView.hh

```diff
@@ -563,10 +559,10 @@ CELER_FUNCTION void VecgeomTrackView::cross_boundary()
                                             vgnext_);
         }
 #else
-        // Some navigators require an lvalue temp_pos
-        auto temp_pos = detail::to_vector(this->pos_);
-        Navigator::RelocateToNextVolume(
-            temp_pos, detail::to_vector(this->dir_), vgnext_);
+        //// Some navigators require an lvalue temp_pos
+        // auto temp_pos = detail::to_vector(this->pos_);
+        // Navigator::RelocateToNextVolume(
+        //     temp_pos, detail::to_vector(this->dir_), vgnext_);
```

Comment:
If this is no longer needed, please delete the `#else` condition. Also try to make a habit of not using `//` for disabling code. `//` is for textual comments, `#if 0` uses the preprocessor to disable code.